### PR TITLE
React save content

### PR DESCRIPTION
**Pull request by phathoang**

### CODE REVIEW

**Comment by phathoang** [comment]
@MattiasNilsson Now saving should work 96a84186

File: Resources/Private/Javascript/components/LoadingOverlay.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }
+}
+
+class LoadingOverlay extends React.Component {
+
+    render () {
+        if (this.props.showLoadingOverlay === true) {
+            return (
+                <div className="t3-frontend-editing__loading-screen" 
+                    dangerouslySetInnerHTML={{__html: FrontendEditing.userIcon}}>
```
Comment (MattiasNilsson) [inline Resources/Private/Javascript/components/LoadingOverlay.js]: The icon is not displayed when loading...

Comment (phathoang) [inline Resources/Private/Javascript/components/LoadingOverlay.js]: Now it is a4d06ef

**Comment by MattiasNilsson** [comment]
Sweet!

**Pull request merged**